webapp/packages/core-blocks/src/index.ts

```diff
@@ -35,8 +35,6 @@ export * from './localization/useTranslate';
 export * from './ConnectionImageWithMask/ConnectionImageWithMask';
 export { default as ConnectionImageWithMaskSvgStyles } from './ConnectionImageWithMask/ConnectionImageWithMaskSvg.m.css';
 
-export * from './Menu/ACTION_ICON_BUTTON_STYLES';
-export { default as ActionIconButtonStyles } from './ActionIconButton.m.css';
```

Comment:
`export { default as ActionIconButtonStyles } from './ActionIconButton.m.css';`
please keep it so we can use it in SContext if needed